rosidl_typesupport_opensplice_cpp/include/rosidl_typesupport_opensplice_cpp/requester.hpp

```diff
@@ -0,0 +1,141 @@
+/* Copyright 2015 Open Source Robotics Foundation, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __rosidl_typesupport_opensplice_cpp__requester__h__
+#define __rosidl_typesupport_opensplice_cpp__requester__h__
+
+#include <atomic>
+#include <limits>
+#include <random>
+
+#include <ccpp_dds_dcps.h>
+
+#include <rosidl_typesupport_opensplice_cpp/message_type_support.h>
+#include <rosidl_typesupport_opensplice_cpp/service_type_support.h>
```

Comment:
Reading up on the difference, it seems that when a `""` search fails it is implicitly converted to `<>` and the search is resumed. On some systems only the `""` style include will search relative to the current file, but that doesn't seem standard. In general relative includes are not recommended anyways.

Basically it seems to not matter, so we can use (abuse?) the lack of distinction to suggest the origin of the header w.r.t. our packaging structure, as @tfoote suggested.